docs/network-flow-visibility.md

```diff
@@ -202,31 +203,56 @@ helm install theia build/charts/theia -n flow-visibility --create-namespace \
+Flow Aggregator Helm Chart values.
 
-```yaml
-apiVersion: v1
-kind: Secret
-metadata:
-  labels:
-    app: flow-aggregator
-  name: clickhouse-secret
-  namespace: flow-aggregator
-stringData:
-  password: clickhouse_operator_password
-  username: clickhouse_operator
-type: Opaque
-```
-
-Please refer to the [Flow Aggregator Configuration](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md#configuration-1)
+Please refer to the Flow Aggregator Helm Chart [README](../build/charts/flow-aggregator/README.md)
+and [Flow Aggregator Configuration](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md#configuration-1)
 to learn about more ClickHouse configuration options in Flow Aggregator.
 
+##### ClickHouse Cluster
+
+From Theia v0.2.0, we support deploying ClickHouse cluster with data replication.
+A ClickHouse cluster consists of shards. Shards refer to the servers that contain
+different parts of the data, which provides a horizontal cluster scaling strategy.
+Each shard consists of one or more replica hosts to ensure reliability.
+
+To deploy ClickHouse cluster, please set `clickhouse.cluster.enable` to true and
+set `clickhouse.cluster.shards` and `clickhouse.cluster.replicas` per your
+requirement.
+
+ClickHouse uses [Apache ZooKeeper](https://zookeeper.apache.org/) for storing
+replicas meta information. By default, a ZooKeeper cluster using `emptyDir` with
+3 replicas will be deployed for ClickHouse cluster. The 3 replicas are expected
+to be deployed on 3 different Nodes. To use a customized ZooKeeper cluster,
+please refer to the
+[ZooKeeper setup instructions for ClickHouse](https://github.com/Altinity/clickhouse-operator/blob/master/docs/zookeeper_setup.md)
+and set `clickhouse.cluster.zookeeperHosts` to your ZooKeeper hosts.
+
+The default affinity allows only one ClickHouse instance per Node. Please make
+sure there are at least 4 Nodes available in your Kubernetes cluster to deploy
+a 2-shard-2-replica cluster with this affinity. To change the affinity, please
+set `clickhouse.cluster.podDistribution` per your requirement.
+
+We recommend using Persistent Volumes if you are going to use ClickHouse cluster
+in production. ClickHouse cluster with in memory deployment is mainly for test
+purpose, as when a Pod accidentally restarts, the memory will be cleared, which
+will lead to a complete data loss. In that case, please refer to the
+[ClickHouse doc](https://clickhouse.com/docs/en/engines/table-engines/mergetree-family/replication/#recovery-after-complete-data-loss)
+to recover the data.
+
+ClickHouse cluster can be deployed with default Local PV or NFS PV by setting
+`clickhouse.storage.createPersistentVolume`. Please make sure the provided path
+contains a folder for each replica with a name in the format `vol<i>`, where `i`
+varies from 0 to the total number of replicas. To have more flexibility in the
+PV creation, we recommend using customized StorageClass by setting
```

Comment:
Do not understand the sentence. Do you mean: "To have more flexibility in the PV creation, you can configure a customized `StorageClass` in `clickhouse.storage.persistentVolumeClaimSpec`?

Reply:
Yes. Updated. Thanks for the rephrasing!